scripts/oes-data-migration-scripts/migration_v3.12.x_to_v4.0.x.py

```diff
@@ -1571,6 +1577,7 @@ def addDBVersion(version):
         # set db version
         date = datetime.datetime.now()        
         data = version,date,date
+        cur_platform.execute("DELETE FROM  db_version (version_no, created_at, updated_at) VALUES (%s, %s, %s)",data)
```

Comment:
Whats this @sudhakaropsmx ? . it should be "DELETE FROM db_version" thats all

Reply:
updated

Reply:
why is it not something like this - DELETE FROM db_version WHERE version_no = version